test/test_torch.py

```diff
@@ -16897,29 +16897,30 @@ def test_exp_slow(self, device, dtype):
         b = torch.exp(torch.ones(1, dtype=dtype, device=device))
         self.assertEqual(a, b.expand(2 ** 31))
 
+    @precisionOverride({torch.bfloat16: 1e-2, torch.float: 0.0002, torch.double: 0.0002})
+    @dtypesIfCUDA(torch.float, torch.double, torch.bfloat16)
     @dtypes(torch.float, torch.double)
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
     def test_hardswish(self, device, dtype):
-        inputValues = [-1000, -4, -3, -2, 0, 2, 3, 4, 1000]
+        inputValues = [-1000, -4, -3, -2, 0, 2, 3, 4]
+        if dtype != torch.bfloat16:
+            inputValues.append(1000)
```

Comment:
what's up with this? Error too large?

Reply:
Yes

Reply:
Do you know if backward for hardswish and hardsigmoid is tested anywhere? Aha, looks like it is tested in test_nn, but only with gradcheck and only for float64. Ok, whatever.